readme.md

```diff
@@ -553,6 +553,7 @@
 - [Homematic](https://github.com/hobbyquaker/awesome-homematic) - Smart home devices.
 - [Ledger](https://github.com/sfischer13/awesome-ledger) - Double-entry accounting on the command-line.
 - [Uncopyright](https://github.com/johnjago/awesome-uncopyright) - Public domain works.
+- [Crypto Currency Tools & Algorithms](https://github.com/kennethreitz/awesome-coins/)
```

Comment:
Can you add a short description here about what crypto currencies are? Not about the list itself. Per guidelines.

How about this?

> Digital currency where encryption is used to regulate the generation of units and verify transfers.

Reply:
done